osm2lanes-web/src/draw.rs

```diff
@@ -107,9 +109,30 @@ pub(crate) fn lanes<R: RenderContext>(
                 rc.draw_text(&layout, (x - (0.5 * font_size), 0.5 * canvas_height));
                 left_edge += width;
             },
-            Lane::Separator { markings, .. } => {
+            Lane::Separator {
+                markings: Some(markings),
+                ..
+            } => {
                 draw_separator(rc, &mut left_edge, markings, &scale, canvas_height);
             },
+            Lane::Separator {
+                markings: None,
+                semantic,
+            } => {
+                if let Some(Semantic::Verge) = semantic {
+                    let width = Metre::new(0.5_f64); // TODO
+                    rc.fill(
+                        Rect::new(
+                            scale.scale(left_edge),
+                            0.0,
+                            scale.scale(left_edge + width),
+                            canvas_height,
+                        ),
+                        &PietColor::GREEN,
+                    );
+                    left_edge += width;
+                }
+            },
```

Comment:
I really like to see this idea. It seems like there should be a default `impl`, and specialisations for different locales. (Would it be worth getting the type system involved in that? Or load from file, like ["schemes" in Map Machine](https://github.com/enzet/map-machine/blob/main/map_machine/scheme/default.yml)?) The different parameters and quirks of the different markings is what gives each style its character and makes it identifiable.

You want to be able to get pretty specific about line drawing, but they can probably be described by a set of cross implementation attributes. Look at the type of thing that the Australian standards (MUTCD type thing) describes:

![image](https://user-images.githubusercontent.com/3784591/172273735-db96e98c-da33-4809-83e7-525e6054dade.png)

Reply:
What about SVG CSS (in yaml?), as a baseline that can be translated into specialised implementations? There are a bunch of [`stroke` attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute#s) that can be set via css in a browser, and do [all sorts of stuff](https://gigacore.github.io/demos/svg-stroke-dasharray-generator/).

Reply:
I think we need a spec that is based on the ground truth and much simpler? From what I can se SVG is more abstract.

All I need is a width in metres, and a `Vec<Metre>` to represent the repeating pattern?

Do we even have any examples where there is a repeating pattern in road paint?

Reply:
Related conversation in https://github.com/a-b-street/osm2streets/issues/24